<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GMSKCCenterReportForm</class>
 <widget class="QDialog" name="GMSKCCenterReportForm">
  <property name="windowTitle">
   <string>Формирование отчёта для ГМСКЦ</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelPeriod">
       <property name="text"><string>Период отчёта:</string></property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="reportPeriod">
       <item><property name="text"><string>Суточный</string></property></item>
       <item><property name="text"><string>Недельный</string></property></item>
       <item><property name="text"><string>Месячный</string></property></item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelContent">
       <property name="text"><string>Содержимое отчёта:</string></property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="reportContent"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="buttonGenerate">
       <property name="text"><string>Сформировать</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text"><string>Закрыть</string></property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>